src/Ombi.Notifications/Agents/EmailNotification.cs

```diff
@@ -73,26 +73,6 @@ private async Task<NotificationMessage> LoadTemplate(NotificationType type, Noti
                 Subject = parsed.Subject,
             };
 
-            if (model.Substitutes.TryGetValue("AdminComment", out var isAdminString))
-            {
-                var isAdmin = bool.Parse(isAdminString);
-                if (isAdmin)
-                {
-                    var user = _userManager.Users.FirstOrDefault(x => x.Id == model.UserId);
-                    // Send to user
-                    message.To = user.Email;
-                }
-                else
-                {
-                    // Send to admin
-                    message.To = settings.AdminEmail;
-                }
-            }
-            else
-            {
-                // Send to admin
-                message.To = settings.AdminEmail;
-            }
```

Comment:
This seems to be dead/redundant code, my tests went fine when removing this. 
See possibly redundant code: https://github.com/Ombi-app/Ombi/pull/4519/files#diff-69436d6a4ec4479e1f5a265b7427ed6fe412d1a060eb8e6275e8574b89cf2c91R135